hackathon_site/dashboard/frontend/src/slices/order/orderSlice.ts

```diff
@@ -56,6 +56,29 @@ export const getTeamOrders = createAsyncThunk<
     }
 });
 
+export const updateOrder = createAsyncThunk<
+    OrderInTable,
```

Comment:
The return type for this thunk is not OrderInTable, the return type is whatever you return (which is response.data --> check swagger to figure out what that is)